### ISSUE TITLE
As a SO/CE I need to make a large table viewable on a mobile device

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/express#1244]
## Context

There are many cases where a site has a data table of information that is not viewable on a mobile device.

If the table contains too many columns or the content of the table causes it to be too wide, the table overflows the content area and is not readable due to the color of text and the body background color.

## Process

1. Login to site as a [SO, CE, EO, ADMIN or DEV]
2. Click Add Content
3. Click Basic Page
4. Copy the html table from the comment below
5. Wrap the table with: [responsive-table][/responsive-table]
6. Click Save

## Expected result

When viewed on a narrow screen or mobile device, the table can be scrolled and not overflow the content area.

## Current result

The table overflows the content area and the table contents are not legible.

![screen shot 2017-07-19 at 8 00 22 am](https://user-images.githubusercontent.com/349713/28370931-c30cd4a8-6c58-11e7-8cf5-5f1713c92021.jpg)

## Expected Result

![screen shot 2017-07-19 at 8 44 13 am](https://user-images.githubusercontent.com/349713/28373033-84bfe9dc-6c5e-11e7-9253-5639b6a269f8.jpg)

## Possible Fix

Adding a shortcode that would wrap a table to provide an overflow area would allow any kind of table to still be viewable.

https://codepen.io/kevincrafts/pen/awgqya

**Comment by alexfinnarn** [comment]
PR for resizing windows for Behat tests https://github.com/CuBoulder/express/pull/752

**Comment by kevincrafts** [comment]
`

Undergraduate | Course | Section | Time | Days | Room | Instructor | Term | ClassLimit
-- | -- | -- | -- | -- | -- | -- | -- | --
MATH 1011 - 3 | Fund. & Tech of College Algebra | 100 | 0915-1050 | MTWRF | KOBL 308 | [icon shape="fa-user-o" /] Little | A | 25
MATH 1011 - 3 | Fund. & Tech of College Algebra | 200 | 1100-1235 | MTWRF | ECCR 151 | [icon shape="fa-user-o" /] Moritz | B | 25
MATH 1012 - 3 | Quant. Reason. Math Skill | 001 | 0900-1200 | MTWRF | ECCR 135 | [icon shape="fa-user-o" /] Angel | M | 25
MATH 1012 - 3 | Quant. Reason. Math Skill | 002 | 1230-0330 | MTWRF | ECCR 110 | [icon shape="fa-user-o" /] Daniel | M | 25
MATH 1012 - 3 | Quant. Reason. Math Skill | 100 | 0915-1050 | MTWRF | MATH 170 | [icon shape="fa-graduation-cap" /] Sparks | A | 25
MATH 1012 - 3 | Quant. Reason. Math Skill | 200 | 0915-1050 | MTWRF | ECCR 151 | [icon shape="fa-user-o" /] Nita | B | 25
MATH 1081 - 3 | Calculus/Soc. Sci. & Bus. | 100 | 1100-1235 | MTWRF | KOBL 102 | [icon shape="fa-user-o" /] Harper | A | 25
MATH 1112 - 4 | Mathematical Analysis in Business | 400 | 0915-1020 | MTWRF | STAD 112 | [icon shape="fa-user-o" /] Grulke | D | 20
MATH 1112 - 4 | Mathematical Analysis in Business | 401 | 1100-1205 | MTWRF | STAD 112 | [icon shape="fa-user-o" /] Grulke | D | 20
MATH 1150 - 4 | Precalculus Mathematics | 400 | 0915-1020 | MTWRF | CLRE 211 | [icon shape="fa-user-o" /] Stalvey | D | 25
MATH 1150 - 4 | Precalculus Mathematics | 401 | 1100-1205 | MTWRF | ECCR 110 | [icon shape="fa-user-o" /] Stalvey | D | 25
MATH 1151 - 1 | Precalculus Supplemental Lab | 400 | 0915-1020 | TH | WAIVING ENROLLMENT | *** | D | 25
MATH 1151 - 1 | Precalculus Supplemental Lab | 401 | 1100-1205 | TH | WAIVING ENROLLMENT | *** | D | 25
MATH 1300 - 5 | Analytic Geometry / Calc 1 | 400 | 0915-1035 | MTWRF | ECCR 116 | [icon shape="fa-graduation-cap" /] Ng / [icon shape="fa-user-o" /] Friedman-Gerlicz | D | 25
Coordinator: Bronstein | Analytic Geometry / Calc 1 | 401 | 0915-1035 | MTWRF | MCOL E158 | [icon shape="fa-graduation-cap" /] Corona / [icon shape="fa-graduation-cap" /] Lu | D | 25
MATH 1300 - 5 | Analytic Geometry / Calc 1 | 402 | 1100-1220 | MTWRF | ECCR 116 | [icon shape="fa-user" /] Bronstein | D | 25
MATH 1300 - 5 | Analytic Geometry / Calc 1 | 403 | 1245-0205 | MTWRF | ECCR 116 | [icon shape="fa-graduation-cap" /] Thompson / [icon shape="fa-graduation-cap" /] Pierson | D | 25
MATH 2001 - 3 | Intro to Discrete Mathematics | 100 | 0915-1050 | MTWRF | DUAN G2B21 | [icon shape="fa-user-o" /] Newberry | A | 25
MATH 2300 - 5 | Analytic Geometry Calc 2 | 400 | 0915-1035 | MTWRF | ECCR 110 | [icon shape="fa-graduation-cap" /] Gerics / [icon shape="fa-graduation-cap" /] Hong | D | 25
Coordinator: Bronstein | Analytic Geometry Calc 2 | 401 | 1100-1220 | MTWRF | KOBL 220 | [icon shape="fa-user-o" /] Roy | D | 25
MATH 2300 - 5 | Analytic Geometry Calc 2 | 402 | 1245-0205 | MTWRF | ECCR 150 | [icon shape="fa-user" /] Bronstein | D | 25
MATH 2300 - 5 | Analytic Geometry Calc 2 | 403 | 1100-1220 | MTWRF | ECCR 150 | [icon shape="fa-graduation-cap" /] Balentine / [icon shape="fa-user-o" /] Newberry | D | 25
MATH 2400 - 4 | Analytic Geometry Calc 3 | 400 | 0915-1020 | MTWRF | ECCR 118 | [icon shape="fa-user-o" /] Mishev | D | 25
Coordinator: Timmer | Analytic Geometry Calc 3 | 401 | 1100-1205 | MTWRF | ECCR 118 | [icon shape="fa-user" /] Timmer | D | 25
MATH 2400 - 4 | Analytic Geometry Calc 3 | 402 | 1245-0150 | MTWRF | ECCR 118 | [icon shape="fa-user-o" /] Roy | D | 25
MATH 2510-3 | Introduction to Statistics | 100 | 1100-1235 | MTWRF | DUAN G2B21 | [icon shape="fa-user-o" /] Daniel | A | 25
MATH 2510-3 | Introduction to Statistics | 200 | 0915-1050 | MTWRF | ECCR 155 | [icon shape="fa-user" /] Timmer | B | 25
MATH 3001-3 | Analysis 1 | 200 | 0915-1050 | MTWRF | DUAN G2B21 | [icon shape="fa-user" /] Trinh | B | 25
MATH 3130 - 3 | Intro to Linear Algebra | 100 | 0915-1050 | MTWRF | KTCH 1B60 | [icon shape="fa-vcard-o" /] Berrizbeitia | A | 25
MATH 3130 - 3 | Intro to Linear Algebra | 200 | 1100-1235 | MTWRF | ECCR 155 | [icon shape="fa-user" /] Manley | B | 25
MATH 3130 - 3 | Intro to Linear Algebra | 201 | 0915-1050 | MTWRF | KOBL 375 | [icon shape="fa-institution" /] Green | B | 25
MATH 3140 | Abstract Algebra 1 | 100 | 0915-1050 | MTWRF | KOBL 102 | [icon shape="fa-institution" /] Green | A | 25
MATH 3430-3 | Ordinary Differential Equations | 200 | 1100-1235 | MTWRF | DUAN G2B21 | [icon shape="fa-vcard-o" /] Berrizbeitia | B | 25
MATH 3510-3 | Intro to Probability & Stats | 100 | 1100-1235 | MTWRF | KOBL 255 | [icon shape="fa-user-o" /] Davison | A | 25
MATH 4510/5510 - 3 | Intro to Probability Theory | 100 | 1245-0220 | MTWRF | MUEN E064 | [icon shape="fa-institution" /] Englander | A | 23
MATH/APPM 4520 | Intro to Mathematical Statistics | 200 | 1245-0220 | MTWRF | KOBL 375 | [icon shape="fa-user-o" /] Yang | B | 25
MATH 4650 - 3 | Intermed Numeric Analysis 1 | 300 | 0300-0520 | MTW | ECCR 150 | [icon shape="fa-graduation-cap" /] APPM | C | 25
MATH 4810/5810 | Special Topics in Mathematics | 050 | 0900-1200 | MTWRF | FLMG 102 | [icon shape="fa-institution" /] Pflaum | G | 20
MATH 5510/4510 - 3 | Intro to Probability Theory | 100 | 1245-0220 | MTWRF | MUEN E064 | [icon shape="fa-institution" /] Englander | A | 4
MATH 5810/4810 | Special Topics in Mathematics | 050 | 0900-1200 | MTWRF | FLMG 102 | [icon shape="fa-institution" /] Pflaum | G | 7
MATH 5810/4810 | Special Topics in Mathematics | *** | 1000-0200 | M-F | MATH 175 | *** | *** | ***

`

**Comment by cathysnider** [comment]
Acceptance Criteria:
- [ ] Tables surrounded by responsive shortcode are wrapped in a div with a horizontal scrollbar at bottom.
- [ ] Table styles continue to work for both responsive and unresponsive tables.
- [ ] The other shortcodes continue to work.

Test site: https://www-dev.colorado.edu/p1ebd7109072/

Test results: Fail on Twitter shortcode.

**Comment by cathysnider** [comment]
Twitter Share continues to not work:
https://www-dev.colorado.edu/p157c24dfe4d/
